test/ActiveRecordFindTest.php

```diff
@@ -277,22 +221,22 @@ public function testFindLast()
 
     public function testFindLastUsingStringCondition()
     {
```

Comment:
Is the old syntax of
$author = Author::find('last', ['conditions' => 'author_id IN(1,2,3,4)']);

going to be supported for backwards compatibility purposes? If so, we need all these old tests, but specify that this old syntax will be deprecated in the future and the new syntax is preferred.

Reply:
> Is the old syntax going to be supported for backwards compatibility purposes? 

No way. It's possible, but we're breaking other parts of the API, and I just don't have the energy to support both, and I don't want to deal with supporting both going forward.

If you feel really strongly about it we can work on it, but definitely not in this PR.